src/Http/Validation/Domain.php

```diff
@@ -53,4 +55,32 @@ protected function stripProtocol($domain)
 
         return $domain;
     }
+
+    /**
+     * Strip the port from a domain.
+     *
+     * @param $domain
+     *
+     * @return mixed
+     */
+    protected function stripPort($domain)
+    {
+        $domainStripped = preg_replace(self::PATTERN_STRIP_PROTOCOL, '', $domain);
```

Comment:
Nevertheless this is the role of the regexp itself to handle this case.
IMO, `preg_match` is more appropriate and a failing test should be added.